README.md

```diff
@@ -1,123 +1,71 @@
-![EmuFlight](docs/assets/images/EmuFlight.png)
+# EgoFlight
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Could not find module `remark-lint-no-unneeded-full-reference-image`](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=5342715)